extension/src/time_series/pipeline/lambda.rs

```diff
@@ -0,0 +1,670 @@
+            f64_lambda_eq!(client, "sin(2.0)",    ( 2.0f64).sin());
+            f64_lambda_eq!(client, "tan(2.0)",    ( 2.0f64).tan());
+            f64_lambda_eq!(client, "sinh(2.0)",   ( 2.0f64).sinh());
+            f64_lambda_eq!(client, "cosh(2.0)",   ( 2.0f64).cosh());
+            f64_lambda_eq!(client, "tanh(2.0)",   ( 2.0f64).tanh());
+            f64_lambda_eq!(client, "asinh(1.0)",  ( 1.0f64).asinh());
+            f64_lambda_eq!(client, "acosh(1.0)",  ( 1.0f64).acosh());
+            f64_lambda_eq!(client, "atanh(1.0)",  ( 1.0f64).atanh());
+
+            f64_lambda_eq!(client, "log(2.0, 10)",   2.0f64.log(10.0));
+            f64_lambda_eq!(client, "atan2(2.0, 10)", 2.0f64.atan2(10.0));
+
+        });
+    }
+
+    #[pg_test]
+    fn test_lambda_unary() {
+        Spi::execute(|client| {
+            client.select("SET timezone TO 'UTC'", None, None);
+            // using the search path trick for this test b/c the operator is
+            // difficult to spot otherwise.
+            let sp = client.select("SELECT format(' %s, toolkit_experimental',current_setting('search_path'))", None, None).first().get_one::<String>().unwrap();
+            client.select(&format!("SET LOCAL search_path TO {}", sp), None, None);
+            client.select("SET timescaledb_toolkit_acknowledge_auto_drop TO 'true'", None, None);
+
+            f64_lambda_eq!(client, "-(2.0)", -2.0f64);
+            f64_lambda_eq!(client, "-(-2.0)", 2.0f64);
+
+            bool_lambda_eq!(client, "not (1 = 1)",  "false");
+            bool_lambda_eq!(client, "not (1 = 2)",  "true");
+            bool_lambda_eq!(client, "not not (1 = 1)",  "true");
+            bool_lambda_eq!(client, "not not (1 = 2)",  "false");
+            bool_lambda_eq!(client, "not (1 <> 1)",  "true");
+            bool_lambda_eq!(client, "not (1 <> 2)",  "false");
+        });
+    }
+
+    #[pg_test]
+    fn test_lambda_interval_ops() {
+        Spi::execute(|client| {
+            client.select("SET timezone TO 'UTC'", None, None);
+            // using the search path trick for this test b/c the operator is
+            // difficult to spot otherwise.
+            let sp = client.select("SELECT format(' %s, toolkit_experimental',current_setting('search_path'))", None, None).first().get_one::<String>().unwrap();
+            client.select(&format!("SET LOCAL search_path TO {}", sp), None, None);
+            client.select("SET timescaledb_toolkit_acknowledge_auto_drop TO 'true'", None, None);
+
+            interval_lambda_eq!(client, "'1 day'i + '1 day'i", "2 days");
+            interval_lambda_eq!(client, "'1 day'i + '1 week'i", "8 days");
+            interval_lambda_eq!(client, "'1 week'i - '1 day'i", "6 days");
+
+            interval_lambda_eq!(client, "'1 day'i * 3", "3 days");
+            interval_lambda_eq!(client, "4 * '1 day'i", "4 days");
+            interval_lambda_eq!(client, "'4 day'i / 4", "1 day");
+        });
+    }
+}
```

Comment:
I'd also like to see a test of variables (i.e. `"let $foo = 5; $foo * $value"`).

Reply:
Huh, I thought I had one of those. Will add.